test/aries-js-worker/test/verifiable/verifiable.js

```diff
@@ -133,6 +133,25 @@ async function verifiableStore(newAries, mode = wasmMode) {
         )
     })
 
+    it(modePrefix + "Alice makes sure that the DID is resolvable", async function () {
+        let success = false
+        for (var i = 0; i < 10; i++) {
```

Comment:
nit: there is a variable for  retries   `let retries = 10;`. Please use it in your test, and also fix it in the original test (from where you copied it).

Reply:
done